client/display/state.rs

```diff
@@ -207,29 +210,25 @@ fn print_table(tasks: &[Task], colors: &Colors, settings: &Settings) {
                 .map(|id| id.to_string())
                 .collect::<Vec<String>>()
                 .join(", ");
-            row.add_cell(Cell::new(text));
+            row.add_cell(style.styled_cell(text, None, None));
         }
 
         if has_labels {
-            if let Some(label) = &task.label {
-                row.add_cell(label.into());
-            } else {
-                row.add_cell(Cell::new(""));
-            }
+            row.add_cell(style.styled_cell(&task.label.as_deref().unwrap_or_default(), None, None));
         }
 
         // Add command and path.
         if settings.client.show_expanded_aliases {
-            row.add_cell(Cell::new(&task.command));
+            row.add_cell(style.styled_cell(&task.command, None, None));
         } else {
-            row.add_cell(Cell::new(&task.original_command));
+            row.add_cell(style.styled_cell(&task.original_command, None, None));
         }
-        row.add_cell(Cell::new(&task.path.to_string_lossy()));
+        row.add_cell(style.styled_cell(&task.path.to_string_lossy(), None, None));
 
         // Add start and end info
         let (start, end) = formatted_start_end(task, settings);
-        row.add_cell(Cell::new(start));
-        row.add_cell(Cell::new(end));
+        row.add_cell(style.styled_cell(start, None, None));
+        row.add_cell(style.styled_cell(end, None, None));
 
```

Comment:
I would rather like to not call `style.styled_cell` on Cells that don't need any styling.

Actually, most of `style.styled_cell` calls in here can be replaced with simple `Cell::new()` calls.
There're also a few in the `log.rs` file.

Reply:
Sure, I can revert that. I had done this because it may be a good idea to consistently _not_ use Cell::new() so you don't, in future, forget to replace these with `style.styled_cell()` calls when you do want to add formatting to those cells.

Reply:
I agree that this might prevent somebody forgetting `style.styled_cell()` at some point, this is something I'll would to take care of when reviewing.

In this case it feels like the benefits of better readability/less obfuscated code surpass the benefit of consistency.
Of course, this is a very subjective opinion based on personal code style ;D